QLTHUVIEN - JPA/src/main/java/com/sun/entity/Histories.java

```diff
@@ -0,0 +1,36 @@
+package com.sun.entity;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+
+import javax.persistence.*;
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+@Entity
+@Table(name = "histories")
+@Getter
+@Setter
+@NoArgsConstructor
+@ToString
+public class Histories implements Serializable {
```

Comment:
```suggestion
public class History implements Serializable {
```
ri chứ em, nhớ sửa tên  file nữa